internal/driver/driver.go

```diff
@@ -65,6 +67,8 @@ func (s *Driver) Start() error {
 // HandleReadCommands triggers a protocol Read operation for the specified device.
 func (s *Driver) HandleReadCommands(deviceName string, protocols map[string]models.ProtocolProperties, reqs []dsModels.CommandRequest) (res []*dsModels.CommandValue, err error) {
 
+	castError := "Failed to parse %v reading, %v"
```

Comment:
```suggestion
        castError := "Failed to parse %s reading, %v"
```
Please use `%s` instead of `%v` as the type of `req.DeviceResourceName` is explicit to be string.

Reply:
Addressed.